travis/shared_configs/python-tester-conda.yml

```diff
@@ -0,0 +1,54 @@
+version: ~> 1.0
+
+jobs:
+  include:
+    - &testpythonconda
+      stage: test
+      name: "Python 3.6"
+      env:
+        - PYTHON_VERSION: 3.6
+      workspaces:
+        use: conda
+      install: skip
+      before_script:
+        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+        - bash miniconda.sh -b -p $HOME/miniconda
+        - export PATH="$HOME/miniconda/bin:$PATH"
+        - conda config --set always_yes yes --set changeps1 no
+        - conda config --remove channels defaults
+        - conda config --add channels conda-forge
+        - conda config --add channels "file://`pwd`/bld-dir"
+        # Useful for debugging
+        - conda info -a
+        - echo "Conda Environment Name':' ${CONDA_ENV_NAME:=testenv}"
+        - echo "Conda Requirements':' ${CONDA_REQUIREMENTS:=dev-requirements.txt}"
+
+        # Manage conda environment
+        - conda create -n ${CONDA_ENV_NAME} python=$PYTHON_VERSION ${CONDA_PACKAGE} ${CONDA_EXTRAS} --file ${CONDA_REQUIREMENTS}
+        - source activate ${CONDA_ENV_NAME}
+        # Useful for debugging
+        - conda list
+      script:
+        - |
+          if command -v coverage; then
+              if [ -f "run_tests.py" ]; then
+                  coverage run --concurrency=thread --parallel-mode run_tests.py
+              else
+                  coverage run --concurrency=thread --parallel-mode -m pytest
+              fi
+              (coverage combine && coverage report | grep -v -e ' 0%') || true
+          else
+              if [ -f "run_tests.py" ]; then
+                  python run_tests.py
+              else
+                  python -m pytest
+              fi
+          fi
+    - <<: *testpythonconda
+      name: "Python 3.7"
+      env:
+        - PYTHON_VERSION: 3.7
+    - <<: *testpythonconda
+      name: "Python 3.8"
+      env:
+        - PYTHON_VERSION: 3.8
```

Comment:
According to the CI output they are working properly (See 3.8 as an example: https://travis-ci.com/github/hhslepicka/typhos/jobs/342458874#L1127). The way that the anchor `testpythonconda` is being used with the `<<` it allow for replacement of sections, in this case the `PYTHON_VERSION`.
Since our recipe is `noarch` there is no need for additional conda builds.